docs/reference/commandline/run.md

```diff
@@ -261,6 +261,38 @@ You may wish to share the UTS namespace with the host if you would like the
 hostname of the container to change as the hostname of the host changes. A more
 advanced use case would be changing the host's hostname from a container.
 
+## <a name="ipc"></a> IPC settings (--ipc)
+
+```text
+--ipc="MODE"  : Set the IPC mode for the container
+```
+
+The `--ipc` flag accepts the following values:
+
+| Value                      | Description                                                                       |
+|:---------------------------|:----------------------------------------------------------------------------------|
+| ""                         | Use daemon's default.                                                             |
+| "none"                     | Own private IPC namespace, with /dev/shm not mounted.                             |
+| "private"                  | Own private IPC namespace.                                                        |
+| "shareable"                | Own private IPC namespace, with a possibility to share it with other containers.  |
+| "container:<_name-or-ID_>" | Join another ("shareable") container's IPC namespace.                             |
+| "host"                     | Use the host system's IPC namespace.                                              |
+
+If not specified, daemon default is used, which can either be `"private"`
+or `"shareable"`, depending on the daemon version and configuration.
+
+IPC (POSIX/SysV IPC) namespace provides separation of named shared memory
```

Comment:
For a follow-up, we should mention `Inter-process communication` on first use of `IPC` to explain the acronym (perhaps we should provide a link to the `ipc(5)` man-page.

Reply:
carried in #4712